README.md

```diff
@@ -1 +1,48 @@
 <!-- TODO: Utilize same structure as the vscode-theme repo with icon on top and descriptive steps -->
```

Comment:
Can we remove this comment aswell? It was only for development

Reply:
sure